docs/site/src/components/apiMenuData.js

```diff
@@ -0,0 +1,68 @@
+// @flow weak
+
+// API menus for components whose entries don't follow `component`, `componentSub1`, `componentSub2` pattern.
+export const apiMenus = {
```

Comment:
Sorry, I don't understand how that's going to work - There's no 1:1 relationship between the three objects, as they cover three different sets of exceptions: non-standard menu content, non-standard component names, and non-standard demo locations. (By non standard I men not following the common pattern.)

While in some cases we use a mapped component name to look up or generate the demo path, we use the same mapped component name to look up or generate the menu, so trying to combine the former would break the latter.